packages/runtime/src/ctx.ts

```diff
@@ -8,10 +8,10 @@ import {
   AccountAddress, AccountStoreI, AlgoTransferParam, ASADeploymentFlags, AssetHoldingM, AssetModFields,
   AssetTransferParam, Context, ExecParams, ExecutionMode,
   SignType, SSCAttributesM, SSCDeploymentFlags, SSCOptionalFlags,
-  State, TransactionType, Txn, TxParams
+  State, TransactionType, Txn, TxParams, UpdateSSCParam
 } from "./types";
 
-const approvalProgram = "approval-program";
+const approvalProgramConst = "approval-program";
```

Comment:
`const approvalProgramConst` doesn't seem right. Maybe we could rename to `const APPROVAL_PROGRAM`?

Reply:
yes